wp-convertkit.php

```diff
@@ -298,7 +298,7 @@ private static function _get_forms($key = null) {
 			$key = is_null($key) ? self::_get_settings('api_key') : $key;
 
 			if(empty($key)) {
-				self::$forms = false;
+				self::$forms = array();
 			} else if(is_null(self::$forms)) {
 				$forms = self::_get_api_response('forms', $key);
```

Comment:
There are still a couple here that will return `false`... let's check those situations and then return either the forms or an empty array?